ethereum-rpc/src/constructors/express/routes/api/v1/tokens.js

```diff
@@ -163,5 +159,101 @@ module.exports = ({
       return res.status(500).json({ errors: [e] });
     }
   });
+
+  router.post("/:address/mint", async (req, res, next) => {
+    try {
+      const { address = "*" } = req.params;
+      let {
+        tokenAddress = "",
+        toAddress = "",
+        id = "",
+        privateKey = "",
+      } = { ...req.query };
+      const tokenContractAddress = prefixContractAddress(
+        tokenAddress.toString("hex")
+      );
+      const recieverAddress = prefixContractAddress(toAddress.toString("hex"));
+      if (!web3.utils.isAddress(address)) {
+        return res.status(500).json({ error: "invalid address" });
+      }
+      if (!web3.utils.isAddress(toAddress)) {
+        return res.status(500).json({ error: "invalid address" });
+      }
+      if (!web3.utils.isAddress(tokenAddress)) {
+        return res.status(500).json({ error: "invalid contract address" });
+      }
```

Comment:
In future PR's let's try adding the `express-validator`, since we are starting to use that pattern.

There is an example in here: `gateway/src/constructors/express/routes/api/v1/tokens/index.js`

Reply:
tried express validator, added to package.json , but not able to access it from the container